### DISCUSSION TITLE
安卓 14 tun 模式问题

### DISCUSSION THREAD

**Discussion by crab21** [MetaCubeX/ClashMetaForAndroid#226] [Q&A]
看起来被 Google ban 了部分 功能，
下面是信息：

Start TUN listening error: create NetworkUpdateMonitor: netlink socket in Android is banned by Google, use the root or system (ADB) user to run sing-box, or switch to the sing-box Adnroid graphical interface client

**Comment by lululoid** [comment]
I need more explanation too about this

**Comment by iannom** [comment]
I have the same problem.
